spec/_attachments/ic.did

```diff
@@ -224,6 +232,17 @@ service ic : {
   bitcoin_send_transaction: (send_transaction_request) -> ();
   bitcoin_get_current_fee_percentiles: (get_current_fee_percentiles_request) -> (vec millisatoshi_per_byte);
 
+  // Canister backup and restore
+  take_snapshot: (record { 
+    canister_id: canister_id;
+  }) -> (snapshot_id);
+  load_snapshot: (record { 
+    canister_id: canister_id;
+    snapshot_id: snapshot_id;
+  }) -> ();
+  list_snapshots: (record {canister_id : canister_id}) -> (vec snapshot);
```

Comment:
```suggestion
  list_snapshots: (record {canister_id : principal}) -> (vec snapshot);
```

Reply:
(assuming @mraszyk suggestions was correct)

Reply:
Actually, we have a type synonym `type canister_id = principal;` at the top of the `ic.did` file so it's fine as is.